webapp-django/crashstats/crashstats/views.py

```diff
@@ -1763,6 +1764,44 @@ def crontabber_state(request, default_context=None):
     return render(request, 'crashstats/crontabber_state.html', context)
 
 
+@utils.json_view
+def crontabber_status(request):
+    api = CrontabberState()
+
+    # start by assuming the status is OK which means no jobs are broken
+    context = {'status': 'OK'}
+
+    all = api.get()['state']
+    broken = [
+        name for name, state in all.items()
+        if state['error_count']
+    ]
+    blocked = [
+        name for name, state in all.items()
+        if set(broken) & set(state['depends_on'])
+    ]
+
+    # This infinite loop recurses deeper and deeper into the structure
+    # to find all jobs that are blocked. If we find that job X is blocked
+    # by job Y in iteration 1, we need to do another iteration to see if
+    # there are jobs that are blocked by job X (which was blocked by job Y)
+    while True:
+        also_blocked = [
+            name for name, state in all.items()
+            if name not in blocked and set(blocked) & set(state['depends_on'])
```

Comment:
Same here, this a bit tricky to understand and I think would be easier to understand with just an ``in``. But maybe I'm actually not understanding it?